Lab_2/nginx/devops-js-app.conf

```diff
@@ -0,0 +1,81 @@
+upstream front_end_app {
+  server localhost:4200; #provide port of your client's app
+}
+
+upstream rest_api_server {
+  server localhost:3000; #provide port of your back-end's app
+}
+
+# configuration for apps starting locally, for debug/development purposes
```

Comment:
Do we need to split different configs into separate files?

Reply:
divided into separate configs